torch/csrc/jit/passes/quantization.cpp

```diff
@@ -88,7 +158,37 @@ void InsertObserverNodes(std::shared_ptr<Graph>& graph, Node* observer_node) {
 }
 
 void InsertFakeQuantNodes(std::shared_ptr<Graph>& graph) {
-  throw std::runtime_error("Pass not implemented yet!");
+  std::stack<Block*> blocks_to_visit;
+  blocks_to_visit.push(graph->block());
+  while (!blocks_to_visit.empty()) {
+    Block* b = blocks_to_visit.top();
+    blocks_to_visit.pop();
+
+    for (Node* n : b->nodes()) {
+      for (Block* subblock : n->blocks()) {
+        blocks_to_visit.push(subblock);
+      }
+
+      if (!checkIfNodeValuesQuantizable(n)) {
```

Comment:
This name is a bit misleading - it looks like the function is going to check values, while in fact it only checks the node type and doesn't look at values.